groups/managers.py

```diff
@@ -83,3 +108,16 @@ class CommentManager(PolymorphicManager, CommentManagerMixin):
     """PolymorphicManager for BaseComments with custom methods."""
     def get_queryset(self):
         return CommentQuerySet(self.model, using=self._db)
+
+
+class ActiveUserQuerySetMixin:
+    """
+    A mixin that adds an active() method, returning users that have recently posted.
+
+    Can be mixed into a Manager or a QuerySet.  The method accepts a parameter for the
+    number of days in the past a user can have posted in order to be considered active,
+    which defaults to `default_active_threshold_days` in `GroupsConfig`.
```

Comment:
I think the part of this that explains the `active` method's interface should be in the `active` method's own docstring.